examples/nlp_seq/models.py

```diff
@@ -79,8 +78,7 @@ class AddPositionEmbs(nn.Module):
 
   @nn.compact
   def __call__(self,
-               inputs,
-               inputs_positions=None):
```

Comment:
wrt packed data : did it ever work?
(if yes: why removing this functionality?)

Reply:
I never tried and never used it basically - it got copied down from other transformer code.